lxd/main_forknet.go

```diff
@@ -107,3 +129,25 @@ func (c *cmdForknet) RunInfo(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func (c *cmdForknet) RunDetach(cmd *cobra.Command, args []string) error {
+	lxdPID := args[1]
+	ifName := args[2]
+	hostName := args[3]
+
+	// Remove all IP addresses from interface before moving to parent netns.
+	// This is to avoid any container address config leaking into host.
+	_, err := shared.RunCommand("ip", "address", "flush", "dev", ifName)
+	if err != nil {
+		return err
+	}
+
+	// Rename the interface, set it down, and move into parent netns.
+	_, err = shared.RunCommand("ip", "link", "set", ifName, "down", "name", hostName, "netns", lxdPID)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("OK")
```

Comment:
Is that left over from debugging?

Reply:
@brauner I consciously left it in just so when running it manually there was some output to show that everything worked OK (as if there is an error the contents if printed to stderr too). Happy to remove it though as it doesn't service any functional purpose when not debugging.